doc-tools/missing-doclet/src/main/java/org/opensearch/missingdoclet/MissingDoclet.java

```diff
@@ -59,12 +60,18 @@ public class MissingDoclet extends StandardDoclet {
     private static final int METHOD = 2;
     // checks that @param tags are present for any method/constructor parameters
     private static final int PARAMETER = 3;
+    private static final Set<String> ALLOWED_TO_BE_IGNORED = new HashSet<>();
```

Comment:
Does `new HashSet<>(List.of("org.opensearch.extensions.proto"));` work or am I being too nitpicky?

Reply:
Yeah, that works.